src/ui/NoNameFacet/NoNameFacetOptions.ts

```diff
@@ -31,16 +42,13 @@ export const NoNameFacetOptions = {
    */
   useAnd: ComponentOptions.buildBooleanOption({ defaultValue: false, section: 'Filtering' }),
   /**
-   * Specifies whether to allow the user to toggle between the `OR` and `AND` modes in the facet.
-   *
-   * Setting this option to `true` displays an icon in the top right corner of the facet. The user can click this icon
-   * to toggle between between the two modes.
-   *
+   * Specifies whether to allow the user to toggle between the **Collapse** and **Expand** modes in the facet.
+   * Default value is `false`.
+   */
+  enableCollapse: ComponentOptions.buildBooleanOption({ defaultValue: false, section: 'Filtering' }),
+  /**
+   * Specifies whether to allow the facet should be in the **Collapse** mode.
    * Default value is `false`.
    */
-  enableTogglingOperator: ComponentOptions.buildBooleanOption({
-    defaultValue: false,
-    alias: 'allowTogglingOperator',
-    section: 'Filtering'
-  })
+  isCollapsed: ComponentOptions.buildBooleanOption({ defaultValue: false, section: 'Filtering' })
```

Comment:
Option name could be a bit misleading perhaps ?

Would `collapsedByDefault` be a better name ?